### ISSUE TITLE
[A11yInspect] Fail: 1.1.1 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#8]
## ❌ Accessibility Fail

**Rule:** 1.1.1 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 1, **Column:** 1

### Description
Missing image description (Critical)

### Context
```html
<img src="/image1.png">
```

### Selector
`html > body > div > div > img:nth-of-type(1)`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
